common_utils/shareMem.go

```diff
@@ -0,0 +1,58 @@
+package common_utils
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	memKey = 7749
+	memSize = 1024
+	memMode = 01600
+)
+
+func SetMemCounters(counters *[len(CountersName)]uint64) error {
+	shmid, _, err := syscall.Syscall(syscall.SYS_SHMGET, uintptr(memKey), uintptr(memSize), uintptr(memMode))
```

Comment:
>SYS_SHMGET [](http://example.com/codeflow?start=42&length=10)

Why you need shared memory? Is it in HLD?

Reply:
I use shared memory to implement gnmi_dump, and it's only used for debug and unit test. 
So, it's not included in HLD.